tools/ci-build/changelogger/README.md

```diff
@@ -1,14 +1,82 @@
 # ChangeLogger
 
-The Changelogger tool generates public facing `.md` changelogs from the structured `CHANGELOG.next.toml` formats that developers modify. The changelogger runs during smithy-rs releases to generate the smithy-rs specific changelog as well as during SDK releases to generate the SDK changelog. The smithy-rs changelog generation moves the AWS changelog entries into a separate file so that they can later be consumed.
+The ChangeLogger tool generates public facing `.md` changelogs from the individual changelog entry Markdown files stored in the `.changelog` directory. The changelogger runs during smithy-rs releases to generate the smithy-rs specific changelog as well as during SDK releases to generate the SDK changelog. The smithy-rs changelog generation duplicates the AWS changelog entries into a separate file so that they can later be consumed.
 
 [smithy-rs-maintainers.txt](./smithy-rs-maintainers.txt) controls the set of users that is **not** acknowledged for their contributions in changelogs.
 
 ## Commands
-### Split
-Splits changelog entries into two `json` files, splitting up `aws-sdk-rust` entries from `smithy-rs` entries. This is a prestep to full changelog generation. The end result is `aws/SDK_CHANGELOG.next.json`.
+### Ls
+Display a preview of pending changelog entries (either for `smithy-rs` or for `aws-sdk-rust`) since the last release to standard output.
+
+This command is intended for developer use.
 
-This command is invoked as part of `smithy-rs-release`.
+### New
+Create a new changelog entry Markdown file in the `.changelog` directory.
+
+This command is intended for developer use.
 
 ### Render
-Render a `.md` format changelog from a structured changelog.
+Render a `.md` format changelog from changelog entry Markdown files stored in the `.changelog` directory.
+
+This command is intended for automation and invoked as part of `smithy-rs-release`.
+
+### Split
+Duplicate `aws-sdk-rust` entries from those in the `.changelog` directory into `aws/SDK_CHANGELOG.next.json`, which is a prestep to full changelog generation.
+
+This command is intended for automation and invoked as part of `smithy-rs-release`.
+
+## Installation
+To install, run the following from this README path:
+```
+$ cargo install --locked --path .
+```
+Confirm the installation:
+```
+$ changelogger --version
+```
+
+## How to Use
+### Ls
+To preview changelog entries for an upcoming release in `smithy-rs`:
+```
+$ changelogger ls --change-set smithy-rs
```

Comment:
Nit: If I have an unparsable changelog file in the directory and use this command, it gives me an error but does not specify which file failed. It would be helpful if the tool could indicate the specific file that caused the issue. For example, this is all it provided:

```
> changelogger ls --change-set smithy-rs
Error: no parsers in chain parsed
---
applies_to: fdahad[]
authors: []
references: []
breaking: false
new_feature: false
bug_fix: false
---

 into `Changelog`
```